### PR TITLE
OCPBUGS-26933: Copy response code from proxied plugin requests

### PR DESCRIPTION
**Pull request by TheRealJon**

### CODE REVIEW

File: pkg/plugins/handlers.go [hunk 1]
```diff
@@ -167,6 +167,9 @@ func (p *PluginsHandler) proxyPluginRequest(requestURL *url.URL, pluginName stri
 	}
 	defer resp.Body.Close()
 
+	// Make sure to copy status code from the plugin service response
+	w.WriteHeader(resp.StatusCode)
```
Comment (KevinFCormier) [inline pkg/plugins/handlers.go:171]: I think this needs to be moved to after line 180, so that the headers can be copied to the response first.

Comment (jhadvig) [reply pkg/plugins/handlers.go:171]: @KevinFCormier is right. This needs to be moved before `io.Copy`

**Review by jhadvig — CHANGES_REQUESTED** [review]

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**